components/script/dom/csskeyframesrule.rs

```diff
@@ -101,6 +103,24 @@ impl CSSKeyframesRuleMethods for CSSKeyframesRule {
             self.rulelist().item(idx as u32)
         }).and_then(Root::downcast)
     }
+
+    // https://drafts.csswg.org/css-animations/#dom-csskeyframesrule-name
+    fn Name(&self) -> DOMString {
+        DOMString::from(&*self.keyframesrule.read().name)
+    }
+
+    // https://drafts.csswg.org/css-animations/#dom-csskeyframesrule-name
+    fn SetName(&self, value: DOMString) -> ErrorResult {
+        // https://github.com/w3c/csswg-drafts/issues/801
+        // Setting this property to a CSS-wide keyword or `none` will
+        // throw a Syntax Error.
+        if value == "initial" || value == "inherit" ||
```

Comment:
This should probably be ascii case-insensitive comparison probably?

But given not even Gecko does this, do we really care?